### DISCUSSION TITLE
Remember maximized state

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#583] [Ideas]
Right now, after connect to server syncplay will always start un-maximized, even if in previous run it's been maximized.

**Comment by Et0h** [comment]
Making it so that Syncplay remembers the maximised state seems like a perfectly good idea to me. 

As far as I can see the two barriers are:
a) the need for someone to volunteer to develop it, and
b) the need for users to then test it - it would need to be tested across multiple platforms, and include at least one test on multi-screen monitors and when the monitor which remembered the position is removed in such circumstances.)

**Comment by soredake** [comment]
Can't describe how annoying it is to maximize it every time....
